static/app/views/performance/traces/content.tsx

```diff
@@ -292,13 +346,13 @@ const StyledPanel = styled(Panel)`
 const TracePanelContent = styled('div')`
   width: 100%;
   display: grid;
-  grid-template-columns: repeat(1, min-content) auto repeat(4, min-content);
+  grid-template-columns: repeat(1, min-content) auto repeat(2, min-content) 120px 66px 10px;
```

Comment:
Do we want to give the breakdown column a fixed width (maybe I'm counting wrong) and give the duration column flexible width?

Reply:
Nah since we need to share the fixed width duration amongst the trace and span table for everything to line up nicely. Both breakdowns self-set their width so while it says `min-content` it's really the same width provided by the element itself.